.kokoro/deps-finder.sh

```diff
@@ -28,175 +28,187 @@ scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
 # include common functions
 source ${scriptDir}/common.sh
 
-#Returns 1 if we find an updated dependency in the commit message.
-function getDependency() {
-  currCommitNumber=$(git rev-parse HEAD)
-
-  #Note that this grep serves to check if we're updating a dependency including shared-dependencies.
-  currCommitMessage=$(git log -1 $currCommitNumber | grep "deps: update dependency com.google.cloud:google-cloud-")
-
-  if [[ -z $currCommitMessage ]]
-  then
-    echo "Commit message does not have correct formatting. Assuming no error."
-    return 1
-  fi
-
-  dependency=${currCommitMessage:28}
-  dependency=${dependency/ to v/:}
-  dependency=${dependency/*com.google.cloud:}
-
-  if [[ -z $dependency ]]
-  then
-    return 1
-  fi
-
-  echo "$dependency"
-  return 0
+#Returns 1 if we find an updated client library in the commit message.
+#Echos client library (if found) before completion.
+function getLibraryUpdate() {
+ currCommitNumber=$(git rev-parse HEAD)
+
+ #Note that this grep serves to check if we're updating a dependency including shared-dependencies.
+ currCommitMessage=$(git log -1 $currCommitNumber | grep "deps: update dependency com.google.cloud:google-cloud-")
+
+ if [[ -z $currCommitMessage ]]
+ then
+   echo "Commit message does not have correct formatting. Assuming no error."
```

Comment:
seems to contradict line 31?

Reply:
This function was added later to a few file 'shared-deps-helper.sh'. This returns 1 because it failed to find a valid commit message, however note that in the checkLibrary() function of 'deps-finder.sh', we return 0 if we're unable to find the correct formatting, since that means the commit being checked doesn't have anything to do with updating dependencies.